### ISSUE TITLE
Problem with text blocks in annotations

### ISSUE THREAD

**Issue by olenagerasimova** [yegor256/qulice#1129]
Qulice fails to process such swagger documentation:
```
@Operation(
        summary = "Get repository settings by name",
        description = """
            java 
            multiline
            text
            block
          """,
        responses = {
            @ApiResponse(
                description = "Returns repository setting json",
                responseCode = "200",
                content = @Content(mediaType = "application/json")
            ),
            @ApiResponse(responseCode = "404", description = "Repository not found")
        }
    )
```
with exception 
```
[ERROR] Failed to execute goal com.qulice:qulice-maven-plugin:0.21.1:check (default-cli) on project artipie: Execution default-cli of goal com.qulice:qulice-maven-plugin:0.21.1:check failed: java.util.concurrent.ExecutionException: java.lang.IllegalStateException: Failed to process files: Exception was thrown while processing /Users/alena/artipie/artipie/src/main/java/com/artipie/api/docs/Repository.java: begin 249, end 15, length 15 -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/PluginExecutionException
авг. 17, 2022 4:05:38 PM net.sourceforge.pmd.RulesetsFactoryUtils getRuleSets
SEVERE: Ruleset not found
net.sourceforge.pmd.RuleSetNotFoundException: Can't find resource 'category/java/multithreading.xml' for rule 'null'.  Make sure the resource is a valid file or URL and is on the CLASSPATH. Here's the current classpath: /Users/alena/apache-maven-3.5.3/boot/plexus-classworlds-2.5.2.jar
        at net.sourceforge.pmd.RuleSetReferenceId.getInputStream(RuleSetReferenceId.java:405)
        at net.sourceforge.pmd.RuleSetFactory.parseRuleSetNode(RuleSetFactory.java:330)
        at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:214)
        at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:209)
        at net.sourceforge.pmd.RuleSetFactory.parseRuleSetReferenceNode(RuleSetFactory.java:489)
        at net.sourceforge.pmd.RuleSetFactory.parseRuleNode(RuleSetFactory.java:448)
        at net.sourceforge.pmd.RuleSetFactory.parseRuleSetNode(RuleSetFactory.java:367)
        at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:214)
        at net.sourceforge.pmd.RuleSetFactory.createRuleSet(RuleSetFactory.java:209)
        at net.sourceforge.pmd.RuleSetFactory.createRuleSets(RuleSetFactory.java:170)
        at net.sourceforge.pmd.RuleSetFactory.createRuleSets(RuleSetFactory.java:154)
        at net.sourceforge.pmd.RulesetsFactoryUtils.getRuleSets(RulesetsFactoryUtils.java:38)
        at net.sourceforge.pmd.processor.AbstractPMDProcessor.createRuleSets(AbstractPMDProcessor.java:68)
        at net.sourceforge.pmd.processor.AbstractPMDProcessor.processFiles(AbstractPMDProcessor.java:101)
        at net.sourceforge.pmd.PMD.processFiles(PMD.java:329)
        at com.qulice.pmd.SourceValidator.validateOne(SourceValidator.java:136)
        at com.qulice.pmd.SourceValidator.validate(SourceValidator.java:102)
        at com.qulice.pmd.PmdValidator.validate(PmdValidator.java:67)
        at com.qulice.maven.CheckMojo$ValidatorCallable.call(CheckMojo.java:236)
        at com.qulice.maven.CheckMojo$ValidatorCallable.call(CheckMojo.java:204)
        at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
        at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
        at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
        at java.base/java.lang.Thread.run(Thread.java:833)

```

**Comment by 0pdd** [comment]
@olenagerasimova 2 puzzles [#1192](https://github.com/yegor256/qulice/issues/1192), [#1193](https://github.com/yegor256/qulice/issues/1193) are still not solved.

**Comment by 0pdd** [comment]
@olenagerasimova the puzzle [#1192](https://github.com/yegor256/qulice/issues/1192) is still not solved; solved: [#1193](https://github.com/yegor256/qulice/issues/1193).

**Comment by k3p7i3** [comment]
@olenagerasimova I tried to reproduce the error, but it didn't work out. Maybe it has already been solved in another commit. If not, i need more information (full java file, foor example) to reproduce the problem.
